### PR TITLE
feature: add "pouch exec"

### PR DESCRIPTION
**Pull request by skoo87**
"exec" is used to execute a new process in container. You can execute a
background process or a interactive process(e.g. bash)

usage:
./pouch exec -ti 12345  /bin/sh
./pouch exec 12345  "/bin/echo hello" ,  there is quotation mark.

Signed-off-by: skoo87 <marckywu@gmail.com>

### CODE REVIEW

**Comment by allencloud** [comment]
Feel so good to read such excellent code. Reviewing...

File: cli/exec.go [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+	Interactive bool
+	Terminal    bool
+	Detach      bool
+}
+
+// Init initializes ExecCommand command.
+func (e *ExecCommand) Init(c *Cli) {
+	e.cli = c
+	e.cmd = &cobra.Command{
+		Use:   "exec",
```
Comment (allencloud) [inline cli/exec.go]: Could you add more details about the command usage, like @Letty5411 mentioned in PR https://github.com/alibaba/pouch/issues/129, https://stackoverflow.com/questions/9725675/is-there-a-standard-format-for-command-line-shell-help-text
```
usage: app_name [options] required_input required_input2
  options:
    -a, --argument     Does something
    -b required     Does something with "required"
    -c, --command required     Something else
    -d [optlistitem1 optlistitem 2 ... ]     Something with list
```

**Comment by skoo87** [comment]
Updated, Now the help info:
Usage:
  pouch exec [container] [flags]

Flags:
  -d, --detach        run process in the backgroud
  -h, --help          help for exec
  -i, --interactive   open container's stdin io
  -t, --tty           allocate a tty device

the format of "pouch exec [container] [flags] " is generated automatically by cobra, seems unable to control it.  @allencloud

**Comment by allencloud** [comment]
LGTM

**Pull request merged**